main.php

```diff
@@ -102,6 +102,31 @@ function fillHeader($header)
                     break;
                 }
 
+
+                if ($parameters["columns_from"] === 'header') {
+                    // ensure all slices have same headers
+                    $headers = [];
+                    $previousHeader = null;
+                    foreach ($subFinder as $slicedFilePart) {
+                        $csv = new Keboola\Csv\CsvFile(
+                            $slicedFilePart->getPathname(),
+                            $manifest["delimiter"],
+                            $manifest["enclosure"]
+                        );
+                        $header = $csv->getHeader();
+
+                        if ($previousHeader !== null) {
+                            if ($header !== $previousHeader) {
+                                throw new \Keboola\Processor\CreateManifest\Exception(sprintf(
+                                    'Slice "%s" of sliced file "%s" must have same header as previous slices',
```

Comment:
The slice %s of the sliced file %s must have the same header as the previous slices?

Reply:
lehounce me mate previousHeader, kdyz je to vlastne firstHeader, ale asi ok

Reply:
Číslo slicu bych do zprávy vůbec nedával, pořadí je nedefinovaný. Stačí obecná hláška, že hlavičky nejsou stejný.

Reply:
Upravil jsem error message:
```
Test "sliced-different-columns"
✓ Execution failed as expected (1)
All slices of the sliced file "/tmp/sliced-different-columns/run-5b009686743da8.71567631/in/tables/sliced" must have the same header ("sample2" is first different).
```

Lepší?

Reply:
skvělý!